packages/react/accordion/src/accordion.tsx

```diff
@@ -219,11 +220,23 @@ interface AccordionImplProps extends PrimitiveDivProps {
    * The language read direction.
    */
   dir?: Direction;
+  /**
+   * When set to true, children of content will remain mounted when content is
+   * collapsed.
+   */
+  keepChildrenMounted?: boolean;
```

Comment:
The convention in radix is to use `forceMount` for such things and it should be closest to the source which is AccordionContent

But AccordionContent already supports all of CollapsibleContent props and by extension also supports the forceMount prop 

https://github.com/radix-ui/primitives/blob/9ae7583f775b868e23c6f05175e095f1d12157fa/packages/react/accordion/src/accordion.tsx#L472

I’m curious why don’t those existing props solve your current problem?